docs/advanced_topics/images/renditions.md

```diff
@@ -68,6 +68,16 @@ By default, Wagtail will try to use the cache called "renditions". If no such ca
 
 When using a queryset to render a list of images or objects with images, you can prefetch the renditions needed with a single additional query. For long lists of items, or where multiple renditions are used for each item, this can provide a significant boost to performance.
 
+## Regenerating existing renditions
+
+You can also directly use the image management command from the console for regenerating the renditions:
```

Comment:
How about:

```suggestion
You can also directly use the image management command from the console to regenerate the renditions:
```
? LGTM otherwise. Thank you @lb-

Reply:
Good call.